app/models/user_permission.py

```diff
@@ -1,30 +1,19 @@
+from dataclasses import dataclass
+
 from app.models import db
 from app.models.base import SoftDeletionModel
 
 
+@dataclass(unsafe_hash=True)
 class UserPermission(SoftDeletionModel):
     """
     User Permissions
     """
     __tablename__ = 'user_permissions'
 
-    id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String, unique=True)
-    description = db.Column(db.String)
-
-    unverified_user = db.Column(db.Boolean)
-    anonymous_user = db.Column(db.Boolean)
-
-    def __init__(self, name, description, unverified_user=False,
-                 anonymous_user=False, deleted_at=None):
```

Comment:
No replacement for this logic

Reply:
Shall I revert this change then?

Reply:
@iamareebjamal - This init will override the default init created by dataclass.